client/src/components/StudentLogin.tsx

```diff
@@ -15,7 +15,7 @@ const LoginForm = () => {
                 initialValues={{ student: '', teacher: '' }}
                 
                 onSubmit={async (values, {setErrors}) => {
-                    await updateStudentTeacher({student: values.student, teacher: values.teacher});
+                    await updateStudentTeacher({student: values.student, teacher: 'me'});
```

Comment:
Can you remove the changes to StudentLogin, a newer version with a bug fix is avaialble on master

Reply:
Remember to rebase on master